scripts/toolchain/windows/build.ps1

```diff
@@ -0,0 +1,114 @@
+# Usage: scripts/toolchain/windows/build.ps1 -ref <ref> -install_prefix <installation directory>
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$ref,
+    [Parameter(Mandatory=$true)]
+    [string]$install_prefix
+)
+
+$ErrorActionPreference = "Stop"
+
+# Detect architecture
+$arch = [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture
+
+# Determine target
+switch ($arch) {
+    x64 {
+        $host_target = "X86"
+    }
+    arm64 {
+        $host_target = "AArch64"
+    }
+    default {
+        Write-Error "Unsupported architecture on Windows: $arch. Only x64 and arm64 are supported."; exit 1
+    }
+}
+
+Write-Host "Building LLVM/MLIR $ref into $install_prefix..."
+
+# Clone LLVM project
+$repo_dir = Join-Path $PWD "llvm-project"
+if (Test-Path $repo_dir) { Remove-Item -Recurse -Force $repo_dir }
+git clone --depth 1 https://github.com/llvm/llvm-project.git --branch $ref $repo_dir
+
+# Change to repo directory
+pushd $repo_dir > $null
+
+# Build LLVM
+try {
+    $build_dir = 'build_llvm'
+    $cmake_args = @(
+        '-S', 'llvm',
+        '-B', $build_dir,
+        '-DLLVM_ENABLE_PROJECTS=mlir',
+        '-DLLVM_BUILD_EXAMPLES=OFF',
+        "-DLLVM_TARGETS_TO_BUILD=$host_target",
+        '-DCMAKE_BUILD_TYPE=Release',
+        '-DLLVM_BUILD_TESTS=OFF',
+        '-DLLVM_INCLUDE_TESTS=OFF',
+        '-DLLVM_INCLUDE_EXAMPLES=OFF',
+        '-DLLVM_ENABLE_ASSERTIONS=ON',
+        '-DLLVM_INSTALL_UTILS=ON',
+        "-DCMAKE_INSTALL_PREFIX=$install_prefix"
```

Comment:
Same questions about LTO and the compiler being used here.
Additionally, this invocation does not enable RTTI as opposed to the others.